testing_and_setup/compass/load_compass_env.sh

```diff
@@ -13,7 +13,8 @@ elif [[ $HOSTNAME = "cooley"* ]]; then
   base_path="/lus/theta-fs0/projects/ccsm/acme/tools/e3sm-unified/base"
 elif [[ $HOSTNAME = "compy"* ]]; then
   base_path="/share/apps/E3SM/conda_envs/base"
-elif [[ $HOSTNAME = "gr-fe"* ]] || [[ $HOSTNAME = "ba-fe"* ]]; then
+elif [[ $HOSTNAME = "gr-fe"* ]] || [[ $HOSTNAME = "ba-fe"* ]] || \
+    [[ $HOSTNAME =~ ^gr[0-9]{4}$ ]] || [[ $HOSTNAME =~ ^ba[0-9]{3}$ ]]; then
```

Comment:
@sbrus89, I marked you as a reviewer because I think this may be a feature you will want to use.  It means you can use the `load_compass_env.sh` script on compute nodes on LANL IC, not just login nodes.

Reply:
@xylar, Nice! That was a great idea.